packages/web-components/src/components/ic-checkbox-group/ic-checkbox-group.tsx

```diff
@@ -26,45 +26,45 @@ export class CheckboxGroup {
   @Element() host: HTMLIcCheckboxGroupElement;
 
   /**
-   * Provide a label for the input.
+   * The label for the input to be displayed.
```

Comment:
```suggestion
   * The label for the checkbox group to be displayed.
```
As we've replaced other occurrences of "input" should we here too?